data/SkyPy/lsst-like_triple_SF.yml

```diff
@@ -89,7 +89,7 @@ tables:
       coefficients: $red.coeff
       magnitudes: $red.M
       filter: bessell-B
-    mag_g, mag_r, mag_i, mag_z, mag_y: !skypy.galaxies.spectrum.kcorrect.apparent_magnitudes
+    mag_*: !skypy.galaxies.spectrum.kcorrect.apparent_magnitudes
```

Comment:
do I understand this correctly that the names of the magnitudes are identical with the full filter names?
Is there a way to have the names the same as before to not break backwards compatibility?